content/utilities/player-list/README.md

```diff
@@ -0,0 +1,6 @@
+# Player List
+
+Shows a list of players that appear on your map.
+Shows information about them (energy/energyCap/silver/planet-count)
+
+Planets outside of your map will not be included, only what you can see.
```

Comment:
I think we should mention the minPlayerPlanetCount and the interval amount as well.

Reply:
```suggestion
Planets outside of your map will not be included, only what you can see and only players with more than two planets. The view will refresh every ten seconds while open.
```